programming-core/src/main/java/org/objectweb/proactive/core/ssh/SshConfig.java

```diff
@@ -135,6 +135,15 @@ private boolean isCapable(SshToken toTest) {
         return false;
     }
 
+    /**
+     * Check if a server-side HostName is defined for the given host
+     * @param host The host associated to the server-side HostName
+     * @return true if a HostName exist for the associated host, false otherwise
+     */
+    private boolean isHostNameDefined(String host) {
+        return getInformation(host, SshToken.HOSTNAME) != null;
```

Comment:
Is it dead code? You should use it below in `getHostName`.

Reply:
ahhh thanks I forgot to remove that dead code, the `getHostName` is doing the check itself